react-node-mongodb/server/db/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Created by Syed Afzal
+ */
+const mongoose = require('mongoose');
+const {log} = require('../utils/helpers/logger');
+
+exports.connect = () => {
```

Comment:
If the MongoDB server isn't up and ready, the server will fail to start
Maybe you can try something like that 
```javascript
exports.connect = (app) => {
    const options = {
        useNewUrlParser: true,
        autoIndex: false, // Don't build indexes
        reconnectTries: 30, // Retry up to 30 times
        reconnectInterval: 500, // Reconnect every 500ms
        poolSize: 10, // Maintain up to 10 socket connections
        // If not connected, return errors immediately rather than waiting for reconnect
        bufferMaxEntries: 0
    }

    const connectWithRetry = () => {
      console.log('MongoDB connection with retry')
      mongoose.connect(process.env.MONGODB_URI, options).then(()=>{
        console.log('MongoDB is connected');
        app.emit('ready');
      }).catch(err=>{
        console.log('MongoDB connection unsuccessful, retry after 2 seconds.')
        setTimeout(connectWithRetry, 2000)
      })
    }
    connectWithRetry();
};
```

Just don't forget to pass the app to the function call to emit the readiness event 😉

Reply:
Done